src/Acuminator/Acuminator.Vsix/Utils/SuppressionFileSystemService.cs

```diff
@@ -28,5 +40,16 @@ public string GetFileName(string path)
 
 			return Path.GetFileNameWithoutExtension(path);
 		}
+
+		public ISuppressionFileWatcherService CreateWatcher(string path)
```

Comment:
@metallmonster  CreateWatcher method is not used currently. Will it be used in the future?

Reply:
@SENya1990 it is used, this is probably a VS bug

Reply:
ok